### DISCUSSION TITLE
default.sex not found

### DISCUSSION THREAD

**Discussion by skycattmll** [astromatic/sextractor#58] [Q&A]
Hi,

I tried to use sextractor and I installed it from conda.

It works fine but I have to copy default.sex, default.param, default.conv into the working directory otherwise it won't work and shows the following:
```
> WARNING: default.sex not found, using internal defaults

----- SExtractor 2.28.0 started on 2023-10-29 at 15:09:21 with 1 thread

> Setting catalog parameters
> *ERROR*: can't read default.param
```

How can I setup the path for sextractor to automatically find these files in its own installation location?

**Comment by kirxkirx** [comment]
I'd argue one shouldn't do that as default.sex (and default.param, default.conv to somewhat lesser extent) are instrument-specific: they include settings that should be optimized for a given telescope+camera combination (also may need to be tweaked depending on how bright your current target is). It make sense to have different sets of these files for different projects...

**Comment by rmathar** [comment]
One might use the `-c` option of the `sex` to specify the full path of the configuration
file `default.sex`, and within that configuration file edit the lines with `FILTER_NAME` to set
the full path name of `default.conf`, and edit `PARAMETERS_NAME` to the full path name of `default.param` .
[I did not test this, but this is what the source code proposes...]
